### PR TITLE
[WIP] Init Arabic translation

### PR DESCRIPTION
**Pull request by nofalx**

### CODE REVIEW

**Comment by nofalx** [comment]
I've signed the individual agreement

File: i18n/ar.toml [hunk 1]
```diff
@@ -0,0 +1,195 @@
+# i18n strings for the English (main) site.
```
Comment (sftim) [inline i18n/ar.toml]:
```suggestion
# i18n strings for the Arabic (العربية) localization.
```
perhaps?

File: content/ar/OWNERS [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+# This is the directory for English source content.
```
Comment (sftim) [inline content/ar/OWNERS]: English?

**Comment by nofalx** [comment]
@salehmashal
@ineskh

**Comment by salehmashal** [comment]
@SuleimanWA

**Comment by remyleone** [comment]
@salehmashal @nofalx Could you:

choose one pull request to move forward and close the other? It would help focus the effort
Ensure that all CLA are signed Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.
rebase on top of the master branch
Ensure that there is no English in the PR. Only arabic.
Let me know if you need any help on those tasks :)

**Comment by remyleone** [comment]
@salehmashal @ineskh @nofalx Don't hesitate to put me (@remyleone ) and @rbenzair as a sponsor. You can find on the following page the template to follow to be included in the organization: https://git.k8s.io/community/community-membership.md#member

**Comment by mohamedsgap** [comment]
@remyleone  @nofalx I can start with you guys 👍

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -179,6 +179,14 @@ aliases:
...
     - devlware
+  sig-docs-ar-owners: # Admins for Arabic content
+    - nofalx
+    - salehmashal
+  sig-docs-ar-reviews: # PR reviews for Arabic content
+    - nofalx
+    - salehmashal
+    - SuleimanWA
+    - ineskh
   sig-docs-vi-owners: # Admins for Vietnamese content
```
Comment (mohamedsgap) [inline OWNERS_ALIASES]:
```suggestion
    - ineskh
    - mohamedsgap
  sig-docs-vi-owners: # Admins for Vietnamese content
```

**Comment by remyleone** [comment]
@salehmashal @nofalx Don't hesitate to put me (@remyleone ) and @rbenzair as a sponsor. You can find on the following page the template to follow to be included in the organization: https://git.k8s.io/community/community-membership.md#member

This PR cannot be merged as long as you are not members of the organization

**Comment by remyleone** [comment]
@salehmashal @nofalx could you rebase and fix the OWNER_ALIASES and README.md ?

**Comment by sftim** [comment]
@nofalx, @salehmashal: any update?

**Comment by mohamedsgap** [comment]
Hey @sftim, You should close this pull request because I actually have one and made so much progress on it, it's only need to be merged

File: OWNERS_ALIASES (hunk 3 above)
Comment (nofalx) [reply OWNERS_ALIASES:190]: add contributers

**Comment by nofalx** [comment]
@remyleone @mohamedsgap 
Unfortunately I dont have enough time to contribute at this current time. I have added @mohamedsgap as an owner in this PR. If you have another PR with better progress you can close this one. Other wise I can transfer the fork into another organization or user. Let me know what you decide on so we can move forward. Thank you!

**Comment by sftim** [comment]
Closed in favor of PR #19758

**Pull request closed without merging**